### ISSUE TITLE
Custom ROMs developers collaboration 

### ISSUE THREAD

**Issue by Micky007** [devmil/PaperLaunch#5]
I was thinking about asking AEX, RR, AOSPA and many other Custom ROMs developers to include your app in their features, I think it really is worth of attention and should substitute any past kind of Apps Sidebar or Circle bar. If they wouldn't give you any kind of credits then I wouldn't request anything.

**Comment by devmil** [comment]
Sounds cool. It has been a while that I was into custom ROMs so I don't know if including a side launcher with a custom ROM is a thing nowadays?

**Comment by Micky007** [comment]
Yes it is ,for example there's app sidebar in Slim Recents on RR.

**Comment by devmil** [comment]
yes, this would definitely be helpful!

**Comment by Micky007** [comment]
Well I've opened a JIRA ticket on DU platforms and on MSM Xtended Sourcefourge, I've posted a feature request on AospExtended,Ground Zero Roms,AOSPA Google+ groups and on RR forum and yesterday and I'm waiting for an answer and AICP "big fish" Ralf Luthier answered me they aren't interested in PaperLaunch integration.
